openslides/global_settings.py

```diff
@@ -11,7 +11,7 @@
 
 AUTHENTICATION_BACKENDS = ('openslides.users.auth.CustomizedModelBackend',)
 
-LOGIN_URL = '/login/'
+LOGIN_URL = '/users/'
```

Comment:
I created #1474 because a TODO in the global_settings can easily be overseen but this has to be done before the 2.0 release.